rotators.h

```diff
@@ -36,12 +36,24 @@
 #define ROTATORS_INCLUDED
 
 #if defined(__WINDOWS__)
-# define ROTATOR_ASCOM
+
+    #define ROTATOR_ASCOM
+    #define ROTATOR_INDI
+
+#elif defined (__APPLE__)
+
+    #define ROTATOR_INDI
+
+#elif defined (__linux__) || defined (__FreeBSD__)
+
+    #define ROTATOR_INDI
+

```

Comment:
let's just define ROTATOR_INDI unconditionally -- it's available on all three platforms PHD2 runs on